expo-plugin/src/android/install_reclaim_project_build_gradle.ts

```
@@ -0,0 +1,22 @@
+import {
+    type ConfigPlugin,
```

ESLint

[error] Delete `··`

(prettier/prettier)